packages/schematics/package.json

```diff
@@ -1,21 +1,42 @@
 {
   "name": "@nrwl/schematics",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
this is a placehoder.

Reply:
We replace it with the real version when building the package.

Reply:
reverted.